src/state_types/msg/actions.rs

```diff
@@ -37,7 +37,7 @@ pub enum ActionSettings {
 #[serde(tag = "userOp", content = "args")]
 pub enum ActionUser {
     Login { email: String, password: String },
-    Register { email: String, password: String },
+    Register { email: String, password: String, tos: bool, privacy: bool, marketing: bool, from: String },
```

Comment:
why is this not in a `gdpr_consent` field?

Reply:
because we create the date in the core

Reply:
and the ui dont care about how data is structured

Reply:
this is overkill; create the date in the UI